install_sct

```diff
@@ -450,6 +473,10 @@ echo -e "Installation type ... $SCT_INSTALL_TYPE"
 echo -e "Operating system .... $OS ($OSver)"
 echo -e "Shell config ........ $RC_FILE_PATH"
 
+# check permissions on shell config file (and fail if necessary)
+echo ""
+check_permissions "$RC_FILE_PATH"
```

Comment:
I was going back and forth on whether we should fail early or just warn. 

Failing early will truly force the user to act, but our current error-handling emits a big scary "Installation failed" message. We want our users to change the permissions first, not go straight to the forum.

(I also worry about the user being locked out from continuing the installation if they *can't* change the permissions on the RC file. But, maybe there's a situation where they would want to proceed and just not edit the RC file.)

Hence using just a warning.